### DISCUSSION TITLE
Planned - Summer holidays with my family during 28 Aug ~ 3 Sep in Singapore!

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#200] [General]
제가 이번 여름 휴가를 6월 26일(일) ~ 7월 2일 (토) 로 계획중입니다.

저도 와이프도 각자 싱가포르를 다녀온지 10년도 넘었고
제가 마리나 베이 샌즈 호텔에 꼭 가고 싶다고
그리고 거기 새로 알게 된 지인 만나러 가고 싶다고 해서 추진 중입니다.

지인 == @wooyaggo86 

@wooyaggo86 님
6월 마지막 주에 싱가포르에서 한번 만날 수 있는 기회가 더 있다면 영광일 것 같습니다.

**Comment by wooyaggo86** [comment]
너무 환영이죠! 구체적인 일정 잡히면 꼭 알려주세요👍👍

**Reply by jongfeel** [reply]
네 알겠습니다!

**Reply by jongfeel** [reply]
6월 마지막주는 아내가 회사 일정이 바쁠 것 같다고 해서 8월 마지막 주로 계획을 잡긴 했습니다.

저희 가족이 보통 여름 휴가를 8월 말 ~ 9월 초 에 계획해서 가기 때문에

계획한 대로 갈 거 같긴 합니다.

변경사항 있으면 또 말씀드릴께요.
